index.html

```diff
@@ -19,8 +19,16 @@
 <meta name="description" content="Adulting...">
 <link rel="shortlink" href="https://git.io/adulting">
 
+<!-- Jam. -->
+<style media="print">
+.hard_candy { font-family: sans-serif }
+.sedate { font-style: inherit }
+.relate { text-decoration-style: wavy }
+.cell { display: block }
+</style>
+
 <!-- Live. -->
-<style title="Pure">
```

Comment:
- switch `ins` underline to `wavy` so that it's like an edit more than a link
- i don't fully understand how `[title]` works